apps/web/src/utils/logger.ts

```diff
@@ -20,16 +26,41 @@ class CustomLogger {
     return CustomLogger.instance;
   }
 
-  private log(level: LogLevel, message: string, meta?: Record<string, unknown>) {
+  private createDatadogLog(level: LogLevel, message: string, meta?: Record<string, unknown>) {
+    let traceId: string | undefined;
+    let spanId: string | undefined;
+
+    //TODO: initialice ddTrace through dd-tracer
+    if (ddTrace) {
+      // Access trace information server-side
+      const currentSpan = ddTrace?.scope().active();
```

Comment:
nit: should this be `ddTrace.scope()?.active();`? namely `ddTrace?` shouldn't be needed since we check above

Reply:
yeah this is true